srml/system/src/lib.rs

```diff
@@ -218,24 +225,35 @@ decl_module! {
 			Self::deposit_event_indexed(&[], event);
 		}
 
+		/// A big dispatch that will disallow any other transaction to be included.
+		// TODO: this must be preferable available for testing really (not possible at the moment).
+		#[weight = SimpleDispatchInfo::MaxOperational]
+		fn fill_block(origin) {
```

Comment:
```suggestion
		fn fill_block(origin) {
			#![cfg(test)]
```

See [this Playground example](//play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=c375a624822567d2e25002ffeae624b1).

Reply:
I don't get the suggestion here; The problem is that:
- `decl_module!{}` does not accept such compiler flags.
- anything that even wraps the internals of the function is useless since the function itself exists and can be called and has a super high weight.
  - The function is not doing anything anyhow.